### DISCUSSION TITLE
【爬虫】开发者急集令:rocket:

### DISCUSSION THREAD

**Discussion by MarkShawn2020** [WeNeedHome/SummaryOfLoanSuspension#950] [Ideas]
之前我已经实现了房天下的一套接口，做到了输入一个城市和楼盘名称，就能返回楼盘的基本信息，并且对接了mongodb数据库。

这个楼盘基本信息的信息量还是很大的，包括经纬度、房价、户数等很多信息，最近讨论区里的资金统计、人口统计等议题都可以基于这套接口的数据进行测算。

但我实在没有太多时间弄了，最近工作上的事情也很紧，所以会的希望主动实现一下，我也可以协助开发！

具体见 [--> python 后端](https://github.com/WeNeedHome/SummaryOfLoanSuspension/blob/main/development/backend/py/README.md)：主要负责开发商与楼盘数据爬虫

**Comment by MarkShawn2020** [comment]
再说明白点，就是接口已经写好了，你在此基础上写个for循环就好了！

当然，肯定也有些脏结果的，稍微改改，问题应该不大，预期至少就算不改代码应该也能拿到30-60%的数据吧！

**Comment by yuhan6665** [comment]
楼主辛苦  有更多的人加入 肯定会越做越好的
我看了一下目前的 CI 和 PR 有一个想法：能不能把目前的 validate 首页数字总和 改成用机器人自动生成 PR 更新首页
这样每个人的 PR 就不用修改数字了 大大降低冲突的机会 还能自动修复一些格式问题

**Reply by MarkShawn2020** [reply]
> 楼主辛苦 有更多的人加入 肯定会越做越好的 我看了一下目前的 CI 和 PR 有一个想法：能不能把目前的 validate 首页数字总和 改成用机器人自动生成 PR 更新首页 这样每个人的 PR 就不用修改数字了 大大降低冲突的机会 还能自动修复一些格式问题

现在的数字，其实本身就是自动生成的，你的想法，只有修改目前的提交机制才可以，比如单独开个省份文件，作为数据源，开发者不要碰readme，readme完全交给ci
